docs/UserGuide.md

```diff
@@ -73,6 +79,20 @@ FindingbrUdders is a **desktop app for managing contacts and scheduling meetings
 
 2. Refer to the [Features](#features) below for details of each command.
 
+<box type="warning">
+
+**IMPORTANT:** Take note that you should **not** edit the _JSON file_ in your directory directly while the app is running or before running.
+</box>
+
+<box type="tip">
+
+### Before you start...
+
+* **anUdder** refers to anyone using the findingbrUdders app and all the contacts in the app
```

Comment:
The term 'anUdder' doesn't seem to be used often. In line 160, we use 'An Udder', so maybe put 'Udder' into the terms instead of 'anUdder' because it is used a lot more often